### PR TITLE
Make age of data available in the UI

### PR DESCRIPTION
**Pull request by ratrun**
Implementation of issue #462.

### CODE REVIEW

File: web/src/main/webapp/js/main.js [hunk 1]
```diff
@@ -1070,15 +1080,19 @@ function routeLatLng(request, doQuery) {
 
             var hiddenDiv = $("<div id='routeDetails'/>");
             hiddenDiv.hide();
+            var versionInfoHiddenDiv = $("<div id='versionDetails'/>");
+            versionInfoHiddenDiv.hide();
```
Comment (karussell) [inline web/src/main/webapp/js/main.js]: Add the version details to the hiddenDiv, then no extra hide() and toggle() call is necessary

**Comment by karussell** [comment]
Thanks! I will have to make a few changes before this also works on GraphHopper Maps though ... but should not take too long.

**Comment by ratrun** [comment]
I have finished an extension providing support for the OSM data date via the `osmosis_replication_timestam` field according to   https://wiki.openstreetmap.org/wiki/PBF_Format#Definition_of_the_OSMHeader_fileblock.

These extensions are here: https://github.com/ratrun/graphhopper/tree/OSMfileHeaderDate. It is probably better to create a separate PR, but if you prefer I can add it to here such that you need to merge only once.

**Comment by karussell** [comment]
Merged this [here](https://github.com/graphhopper/graphhopper/commit/f668788fc750d06f49556cb63caad57ac78ca72b). Please create a new PR for the other nice work!
(The work looks really good, thanks! I would directly return the osmDataDate and reuse the `formatDateTime` method for formatting)

**Comment by karussell** [comment]
This is now live. Currently we use 2 JVMs but there could be more. So I introduced a semi-official json format for the info page ala:
`"car":{"import_date": ...}, "motorcycle":{...}`

and I simply pick the car profile which could be different to others:

`https://graphhopper.com/api/1/info?debug=true&key=[YOUR_KEY]`

**Pull request closed without merging**